webservices/tasks/download.py

```diff
@@ -192,6 +192,6 @@ def export_query(path, qs):
 @app.task
 def clear_bucket():
     for obj in task_utils.get_bucket().objects.all():
-        if not obj.key.startswith('legal') or not obj.key.startswith('bulk-downloads'):
+        if not obj.key.startswith('legal') and not obj.key.startswith('bulk-downloads'):
```

Comment:
There's a better way to go about this I think.  Python's [`str.startswith`](https://docs.python.org/3/library/stdtypes.html#str.startswith) method can also take in a `tuple` of strings to compare and if any of them match, it'll return `True`.  Example:

```python
matches = ('legal', 'bulk-downloads',)

if obj.key.startswith(matches):
    print('The object name starts with one of the matching strings.')
else:
    print('The object does not start with one of the matching strings.')
```

So, we could rewrite this by creating a `tuple` (`set`) of strings to match and then just letting the `startswith` method do the work for us:

```python
matches = ('legal', 'bulk-downloads',)

if not obj.key.startswith(matches):
    obj.delete()
```

That should be easier to read and extend in the future if need be. :-)